src/app/app.component.ts

```diff
@@ -66,7 +66,8 @@ import {ItemComponent} from './dspace/components/item.component';
         { path: "/communities/:id/:page", name: "Communities", component: CommunityComponent },
         { path: "/collections/:id", name: "Collections", component: CollectionComponent },
         { path: "/collections/:id/:page", name: "Collections", component: CollectionComponent },
-        { path: "/items/:id", name: "Items", component: ItemComponent }
+        { path: "/items/:id", name: "Items", component: ItemComponent },
+        { path: "/items/:id/:page", name: "Items", component: ItemComponent }
```

Comment:
I was planning on removing that an modifying the breadcrumb component. This does however leave the option for different views under a item view. Which may be useful, so I left it in.